plugins/uefi-capsule/fu-uefi-cod-device.c

```diff
@@ -173,13 +173,27 @@ fu_uefi_cod_device_get_indexed_filename(FuUefiDevice *self, GError **error)
 static gchar *
 fu_uefi_cod_device_get_filename(FuUefiDevice *self, GError **error)
 {
+	FuContext *ctx = fu_device_get_context(FU_DEVICE(self));
 	g_autofree gchar *esp_path = fu_volume_get_mount_point(fu_uefi_device_get_esp(self));
 	g_autofree gchar *basename = NULL;
+	const gchar *manufacturer = NULL;
 
 	/* InsydeH2O */
 	if (fu_device_has_private_flag(FU_DEVICE(self), FU_UEFI_DEVICE_FLAG_COD_INDEXED_FILENAME))
 		return fu_uefi_cod_device_get_indexed_filename(self, error);
 
+	/* Dell Inc. */
+	manufacturer = fu_context_get_hwid_value(ctx, FU_HWIDS_KEY_MANUFACTURER);
+	if (g_strcmp0(manufacturer, "Dell Inc.") == 0) {
```

Comment:
Is it not possible to use the ACPI UEFI firmware to detect and set a quirk by GUID like we do for Insyde instead?

Reply:
the `UEFI` table does not present, I can't get this information from /sys/fw.

Reply:
It's not in /sys/firmware/acpi/tables?

Reply:
nah, no idea why it is not there.

Reply:
Well that's too bad. This approach you do now will work for most Dell systems, but anything rebranded could break. Not sure if it's worth designing quirk around that though.

There is an SMBIOS string called "Dell System" I seem to recall. Can you flag on that using FuSmbios maybe?